docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1258,6 +1258,45 @@ Renames an existing field. If the field doesn't exist or the new name is already
 }
 --------------------------------------------------
 
+[[script-processor]]
+=== Script Processor
+
+Script Processor allows inline, stored, and file scripts to be executed within Ingest Pipelines.
+
+Scripts use Elasticsearch's Script Service to compile the processor's script for each document.
```

Comment:
Change to:  The script processor use Elasticsearch’s script service to compile the script for each document.